docs/pages/guides/using-push-notifications-services.mdx

```diff
@@ -65,6 +65,20 @@ For implementation details, see the following guides:
   Icon={BookOpen02Icon}
 />
 
+## CleverTap
```

Comment:
Please add this section after Customer.io. Our preferred sorting for this page is to add new sections after the existing ones.